view/frontend/web/js/view/payment/method-renderer/cardknox-method.js

```diff
@@ -13,12 +13,16 @@ define(
         'Magento_Checkout/js/model/full-screen-loader',
         'Magento_Checkout/js/action/place-order',
         'Magento_Ui/js/model/messageList',
-        'Magento_Vault/js/view/payment/vault-enabler'
+        'Magento_Vault/js/view/payment/vault-enabler',
+        'ko',
     ],
-    function (Component, $, v, i,fullScreenLoader,placeOrderAction,messageList,VaultEnabler) {
+    function (Component, $, v, i,fullScreenLoader,placeOrderAction,messageList,VaultEnabler, ko) {
         'use strict';
 
         return Component.extend({
+            cardNumberIsValid: ko.observable(false),
+            cvvIsValid:  ko.observable(false),
```

Comment:
@mstein-ck see the below link for how this works
https://alanstorm.com/knockout-observables-for-javascript-programmers/